File: twoSum.go
func twoSum(nums []int, target int) []int {
    answer := []int{}
    main:
    for i, num1 := range nums {
        for j, num2 := range nums {
            if i == j {
                continue
            }
            tmp_target := num1 + num2
            if target == tmp_target {
                answer = []int{i, j}
                break main
            }
        }
    }
    return answer
}

File: interviews/python/twoSum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        for i in range(len(nums)):
            hashmap[nums[i]] = i

        for i in range(len(nums)):
            rest = target - nums[i]
            if rest in hashmap and hashmap[rest] != i:
                return [i, hashmap[rest]]
